js/js_test.go

```diff
@@ -467,7 +467,7 @@ func TestCallWithNull(t *testing.T) {
 func TestReflection(t *testing.T) {
 	o := js.Global.Call("eval", "({ answer: 42 })")
 	if reflect.ValueOf(o).Interface().(*js.Object) != o {
-		t.Fail()
+		t.Fatal()
```

Comment:
It's usually easier to do these minor things as you're working in a relevant context than later. But this is completely up to you, and not critical for getting the PR into a mergeable state.